components/collator/src/elements.rs

```diff
@@ -1237,11 +1237,7 @@ where
                 }
             }
         };
-        let start = if c.decomposition_starts_with_non_starter() {
-            0
-        } else {
-            1
-        };
+        let start = c.decomposition_starts_with_non_starter() as usize;
```

Comment:
Revert this?

Reply:
I prefer `as usize` over `if {} else {}`

Reply:
I'm still not a fan of bool-to-integer conversions, but it's fine.